frontend/src/main/components/OurTable.js

```diff
@@ -66,7 +68,7 @@ export default function OurTable({ columns, data, testid = "testid", ...rest })
                             {row.cells.map((cell, _index) => {
                                 return (
                                     <td
-                                        {...cell.getCellProps()}
+                                        {...cell.getCellProps({})}
```

Comment:
Do the curly braces within the parameters for `getCellProps` serve a purpose or are they a leftover from an undone change?

Reply:
undone change reverted